src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -214,14 +217,7 @@
                              :on-focus       (fn [] (when @scroll (js/setTimeout #(.scrollToEnd @scroll) 100)))
                              :on-change-text #(re-frame/dispatch [:wallet.send/set-and-validate-amount %])}}
             (when modal?
```

Comment:
What happens if it isn't modal? Why not disabled true for both?

Reply:
GH diff makes it difficult to read because 2 `view`s have been swapped. It's actually the same thing but couple lines above.